RELEASE_NOTES.md

```diff
@@ -13,6 +13,11 @@ Search Service MKII release notes
 * In KBaseSearchEngine.spec, moved the flags addNarrativeInfo and addAccessGroupInfo
```

Comment:
This code really needs a refactor / rewrite, it's too hard to grok. I can't tell if it's right or not because the input constraints are too loose.

Reply:
Same as above. The comment references an unrelated release note, so I'm not following.

Reply:
I couldn't agree me. I have never struggled this much to fix code.